inventory/all_projects/sendmail

```diff
@@ -0,0 +1,3 @@
+
+[libpony_staging]
```

Comment:
What is the difference between the `libpony_staging` group and the `postfix_staging` group? Do we need both for the same server? Or did you mean to put the old server here, maybe?